openedx/core/djangoapps/content/block_structure/config/__init__.py

```diff
@@ -10,12 +10,41 @@
 from .models import BlockStructureConfiguration
 
 # Switches
+# .. toggle_name: INVALIDATE_CACHE_ON_PUBLISH
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Invalidates block structure cache when changes to courses are published.
+# .. toggle_use_cases: opt_in
+# .. toggle_creation_date: 2017-02-23
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/14571
 INVALIDATE_CACHE_ON_PUBLISH = WaffleSwitch(
     "block_structure.invalidate_cache_on_publish", __name__
 )
+
+# .. toggle_name: STORAGE_BACKING_FOR_CACHE
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Enables retrieving block structures from storage instead of regenerating
+#   block structures when they aren't available in cache.
+# .. toggle_warnings: Depends on the storage class, `STORAGE_CLASS`, and storage keyword arguments,
+#   `STORAGE_KWARGS`, specified under `BLOCK_STRUCTURES_SETTINGS`.
+# .. toggle_use_cases: opt_in
+# .. toggle_creation_date: 2017-02-23
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/14512
 STORAGE_BACKING_FOR_CACHE = WaffleSwitch(
     "block_structure.storage_backing_for_cache", __name__
 )
+
+# .. toggle_name: RAISE_ERROR_WHEN_NOT_FOUND
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Raises an error if the requested block structure does not exist in block
+#   structure store, or if it is outdated. Block structure store refers to both cache and storage,
+#   if enabled.
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2017-02-23
+# .. toggle_target_removal_date: 2022-01-01
```

Comment:
Since this is a temporary switch, as [mentioned earlier](https://github.com/edx/edx-platform/pull/26175/files#r564816707), I had to set a removal date. However, I'm in no seat to choose that removal date.

So, @robrap do you have any suggestions on what to do in this case?

Reply:
Answered in https://github.com/edx/edx-platform/pull/26175#discussion_r564842861